packages/api/src/lib/granule-delete.ts

```diff
@@ -85,6 +99,21 @@ const deleteGranuleAndFiles = async (params: {
       { granule_cumulus_id: pgGranule.cumulus_id }
     );
 
+    if (collectionCumulusId) {
+      const collectionPgRecord = await collectionPgModel.get(
+        knex, { cumulus_id: collectionCumulusId }
+      );
+      granuleToPublishToSns = await translatePostgresGranuleToApiGranule({
```

Comment:
I manually do the lookup before hand because, as a TS file, it doesn't allow me to provide no value for `collectionPgRecord`:
```
Argument of type '{ granulePgRecord: PostgresGranuleRecord; knexOrTransaction: Knex<any, unknown[]>; collectionPgModel: CollectionPgModel; filePgModel: FilePgModel; pdrPgModel: PdrPgModel; providerPgModel: ProviderPgModel; }' is not assignable to parameter of type '{ granulePgRecord: PostgresGranuleRecord; collectionPgRecord: PostgresCollectionRecord; knexOrTransaction: Knex<any, unknown[]> | Transaction<...>; collectionPgModel: CollectionPgModel; pdrPgModel: PdrPgModel; providerPgModel: ProviderPgModel; filePgModel: FilePgModel; }'.
```